packages/stacked_generator/lib/src/generators/router/generator/router_class_generator.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:stacked_generator/src/generators/base_generator.dart';
+import 'package:stacked_generator/utils.dart';
+
+import '../route_config/route_config.dart';
+import '../router_config/router_config.dart';
+import 'route_generator_helper.dart';
+
+class RouterClassGenerator extends BaseGenerator with RouteGeneratorHelper {
+  final RouterConfig _rootRouterConfig;
+
+  RouterClassGenerator(this._rootRouterConfig);
+  @override
+  String generate() {
+    writeLine("// ignore_for_file: public_member_api_docs");
+    generateImports(
+        [..._rootRouterConfig.routes, ..._rootRouterConfig.subRouters]);
+    generateRouterByRouter(
+        _rootRouterConfig.routes,
+        _rootRouterConfig.routerClassName,
+        _rootRouterConfig.routesClassName,
+        _rootRouterConfig.generateNavigationHelper);
+    return stringBuffer.toString();
+  }
+
+  void generateRouterByRouter(
```

Comment:
What does this function do? I'd like it to have a better name.

Reply:
This function is created to generate the routes and their subroutes recursively